AuxTel/AuxTel-Troubleshooting/General-Troubleshooting/AT-Lost-Pointing-Accuracy.rst

```diff
@@ -0,0 +1,136 @@
+
+.. This section should include the procedure. There is no strict formatting or structure required for procedures. It is left to the authors to decide which format and structure is most relevant.
+.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
+.. For highly complicated procedures, consider breaking them into separate procedures. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+
+This paragraph describes some general guidance for this procedure. This paragraph is optional depending on its usefulness and the complexity of the procedure.
+
+In the case of a very straightforward procedure, a simple numbered list could be used, like as follows:
+
+.. warning::
+    For this example, this step is critical.
+
+.. _AT-Lost-Pointing-Accuracy-Procedure-Step-1:
+
+If the bright CWFS or spectroscopic target is visible in the detector, follow these steps:
+
+#. Pause the ATScriptQueue.
+
+#. Add a `correct_pointing` script to the queue and move it to the top of the waiting scripts. The configuration could be the default, so leave the configuration empty (AZ: 90, EL: 60) or you can choose an area (AZEL) of the sky by introducing your own configuration or a wider search by modifying the `mag_limit`, `mag_range`, or the search `radius`.
+
+    .. code-block:: python
+
+        az: 180
+        el: 35
+        mag_limit: 6.0
+        mag_range: 4.0
+        radius: 5.0
+
+#. Play the ATQueue and while the script is running, confirm in the incoming RubinTV images that the right target is being centered.
+
+#. If the pointing is lost again, repeat steps 1-3 with a different configuration, different location of the sky, or wider search to ensure the target is appropriate for the `correct_pointing` script.
+
+If the pointing is completely lost and no bright star is seen in the image, follow these steps:
+
+#. Pause ATScriptQueue.
+
+#. Recycle ATPtg by sending it to STANDBY, and then back to ENABLED. This will erase all absorbed zero-point pointing offsets from before.
+
+#. Add a `correct_pointing` script to the queue and move it to the top of the waiting scripts. The configuration could be the default, so leave the configuration empty (AZ: 90, EL: 60) or you can choose an area (AZEL) of the sky by introducing your own configuration or a wider search by modifying the `mag_limit`, `mag_range`, or the search `radius`.
+
+    .. code-block:: python
+
+        az: 180
+        el: 55
+        mag_limit: 8.0
+        mag_range: 4.0
+        radius: 5.0
+
+#. Play the ATQueue and while the script is running, confirm in the incoming RubinTV images that the right target is being centered.
```

Comment:
confirm in the incoming ----- > confirm the incoming

Reply:
done